packages/salesforcedx-vscode-replay-debugger/package.nls.json

```diff
@@ -7,5 +7,6 @@
     "Path to an Apex Debug Log file (*.log) in the workspace folder.",
   "stop_on_entry_text":
     "Stop the debugger after launching. If false, the debugger will run until it hits a breakpoint.",
-  "trace_text": "Output debug log lines as they are processed by the debugger."
+  "trace_text":
+    "Produce diagnostic output. Instead of setting this to true you can list one or more selectors separated with commas. The 'all' selector enables very detailed output. Other selectors are 'protocol', 'logfile', 'launch', 'breakpoints'."
```

Comment:
This text is similar to the one in Apex Interactive Debugger extension.

Reply:
Should the message should mention that setting the trace to `true` defaults to all?

Reply:
I was thinking of just removing the boolean option so it only takes a string. `all` is basically true; nothing specified is basically false. @guw do you remember why the interactive debugger extension supports boolean and string?

Reply:
From an old PR, we have these two types because "This is how the VS Code node debugger does it so I would prefer re-using existing concepts from VS Code for consistency."

For comparison purpose, VS Code's node debugger uses this description:
`Produce diagnostic output. Instead of setting this to true you can list one or more selectors separated with commas. The 'verbose' selector enables very detailled output.`

@ruthemmanuelle , do you think our text needs to be updated?

Reply:
Yeah, this is kind of confusing. How about something like this?

"Produce diagnostic output. You can set this to true, false, or one or more selectors separated with commas. Selectors include 'all' (very detailed output; the equivalent of true), 'protocol', 'logfile', 'launch', 'breakpoints'."